diff --git a/node_modules/cli-table3/src/utils.js b/node_modules/cli-table3/src/utils.js
index c922c5b..290d9f3 100644
--- a/node_modules/cli-table3/src/utils.js
+++ b/node_modules/cli-table3/src/utils.js
@@ -1,4 +1,4 @@
-const stringWidth = require('string-width');
+const stringWidth = import('string-width');
 
 function codeRegex(capture) {
   return capture ? /\u001b\[((?:\d*;){0,5}\d*)m/g : /\u001b\[(?:\d*;){0,5}\d*m/g;
